test/client/fixtures/runner/api/click-test.js

```diff
@@ -1046,35 +1048,36 @@ $(document).ready(function () {
         );
     });
 
-    asyncTest('testcafe functions should not be in strict mode (GH-258)', function () {
-        var exceptionRaised = false;
+    // TODO: uncomment this test after update hammerhead version to 7.0.0
```

Comment:
I suppose we should create an issue `Update to the testcafe-hammehead v7.0.0` and add this comment in the issue.